### DISCUSSION TITLE
How to set stochiometry window in the model?

### DISCUSSION THREAD

**Discussion by xrfe** [pybamm-team/PyBaMM#3338] [Q&A]
Hi,
Let say I have experimental data for anode and cathode stochiometry windows, e.g., 0.01 < x < 0.95, and 0.2 < y < 0.85. where can I set these values in the pybamm model?

**Comment by brosaplanella** [comment]
This is not yet supported, but we are working on it: https://github.com/pybamm-team/PyBaMM/issues/2553

You could set the windows based on voltage though, and charge/discharge until that voltage limit.
